doc/nrf/ug_radio_coex.rst

```diff
@@ -71,19 +75,79 @@ To enable CX support for the currently supported CX implementation, you must add
 You can also provide the node using the devicetree source file of the target board or an overlay file.
 See :ref:`zephyr:dt-guide` for more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences between DTS and Kconfig.
 
-For the CX implementation currently supported, the ``nrf_radio_coex`` node has the ``compatible`` property set to ``generic-radio-coex-three-wire``.
-Devicetree nodes compatible with ``generic-radio-coex-three-wire`` can be used when a three-wire interface to the PTA is provided (like the one shown in the following image).
+The following CX hardware interfaces are currently supported:
+
+* :ref:`ug_radio_cx_nrf700x_hw_if`
+* :ref:`ug_radio_cx_generic_3_wire_hw_if`
+
+Both supported hardware interfaces can be used when a three-wire interface to the PTA is provided (like the one shown in the following image).
 However, the role of each of the pins is dependent on the Wi-Fi Coexistence implementation used.
 
 .. note::
    * When using one of the supported implementations, you must use the ``nrf_radio_coex`` name for the node.
      However, if you add a custom user implementation, you can also use a different name.
-
-   * You can add a new device binding and use it as the ``compatible`` property for the node, if ``generic-radio-coex-three-wire`` is unsuitable.
+   * You can add a new device binding and use it as the ``compatible`` property for the node, if the provided hardware interfaces are unsuitable.
 
 .. figure:: images/coex_generic_3pin_pta.svg
    :alt: PTA interface supported by the ``nrf_radio_coex`` node with an nRF52 Series SoC
 
+
+.. _ug_radio_cx_nrf700x_hw_if:
+
+nRF700x Coexistence hardware interface
+======================================
+
+To configure the hardware for a PTA inside the nRF700x series chips:
+
+1. Add the following node in the devicetree source file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_coex: nrf7002-coex {
+               status = "okay";
+               compatible = "nordic,nrf700x-coex";
+               req-gpios =     <&gpio0 24 (GPIO_ACTIVE_HIGH)>;
+               pri-dir-gpios = <&gpio0 14 (GPIO_ACTIVE_HIGH)>;
+               grant-gpios =   <&gpio0 25 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+         };
+      };
+
+#. Optionally replace the node name ``nrf7002-coex`` with a custom one.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``req-gpios`` - GPIO characteristic of the device that controls the ``REQUEST`` signal of the PTA.
+   * ``pri-dir-gpios`` - GPIO characteristic of the device that controls the ``PRIORITY`` signal of the PTA.
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
```

Comment:
When this comment is addressed https://github.com/nrfconnect/sdk-nrf/pull/8750/files#r972850052
there will probably a need to rename pins (`...that controls the PRIORITY`)

Reply:
Fixed